steward_abi/AaveV2StablecoinCellar.json

```diff
@@ -1792,206 +1780,8 @@
6140fa613eef565b039392505050565b600081518084526020808501945080840160005b8381101561413b5781516001600160a01b031687529582019590820190600101614116565b509495945050505050565b85815284602082015260a06040820152600061416560a0830186614102565b6001600160a01b0394909416606083015250608001529392505050565b6000602080838503121561419557600080fd5b82516001600160401b03808211156141ac57600080fd5b818501915085601f8301126141c057600080fd5b8151818111156141d2576141d2613a53565b8060051b91506141e3848301613ad6565b81815291830184019184810190888411156141fd57600080fd5b938501935b8385101561421b57845182529385019390850190614202565b98975050505050505050565b6000808335601e1984360301811261423e57600080fd5b8301803591506001600160401b0382111561425857600080fd5b60200191503681900382131561426d57600080fd5b9250929050565b8183823760009101908152919050565b60006020828403121561429657600080fd5b81516001600160401b03808211156142ad57600080fd5b818401915084601f8301126142c157600080fd5b8151818111156142d3576142d3613a53565b6142e6601f8201601f1916602001613ad6565b91508082528560208285010111156142fd57600080fd5b61430e81602084016020860161398e565b50949350505050565b60006001820161432957614329613eef565b5060010190565b6060815260006143436060830186614102565b6020830194909452506001600160a01b0391909116604090910152919050565b600181815b8085111561439e57816000190482111561438457614384613eef565b8085161561439157918102915b93841c9390800290614368565b509250929050565b6000826143b557506001610f84565b816143c257506000610f84565b81600181146143d857600281146143e2576143fe565b6001915050610f84565b60ff8411156143f3576143f3613eef565b50506001821b610f84565b5060208310610133831016604e8410600b8410161715614421575081810a610f84565b61442b8383614363565b806000190482111561443f5761443f613eef565b029392505050565b6000610f1660ff8416836143a6565b600060ff821660ff84168082101561447057614470613eef565b90039392505050565b80516fffffffffffffffffffffffffffffffff81168114613df457600080fd5b8051613df481613a12565b8051613df481613e2e565b6000806000806000806000806000806000806101808d8f0312156144d257600080fd5b8c519b506144e260208e01614479565b9a506144f060408e01614479565b99506144fe60608e01614479565b985061450c60808e01614479565b975061451a60a08e01614479565b965060c08d015164ffffffffff8116811461453457600080fd5b955061454260e08e01614499565b94506145516101008e01614499565b93506145606101208e01614499565b925061456f6101408e01614499565b915061457e6101608e016144a4565b90509295989b509295989b509295989b565b6000602082840312156145a257600080fd5b8151610f1681613e2e565b600080835481600182811c9150808316806145c957607f831692505b602080841082036145e857634e487b7160e01b86526022600452602486fd5b8180156145fc57600181146146115761463e565b60ff198616895284151585028901965061463e565b60008a81526020902060005b868110156146365781548b82015290850190830161461d565b505084890196505b50949897505050505050505056feddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3efa2646970667358221220ea6498ab16f3f2edc730577a9653186b5a6461655c1a554ed9f137096534cbd164736f6c634300080f0033536f6d6d656c696572204161766520563220537461626c65636f696e2043656c6c6172204c5020546f6b656e",
```

Comment:
The encoded elements in these two contract calls seem to be of different types, the former has `bytecode: { object: "..." }` and here `bytecode: "..."`. What was the substantive difference in the way these were generated?

Reply:
This was generated using the recommended command from the README for the repo:

```bash
npm run setup
```

Looking through my history I had trouble figuring out how I had generated it in the past. I think I had done it directly with `solc` but I was unable to this time because of imports using hardhat resource paths like `"@solmate/tokens/ERC20.sol"`. The most substantial difference is that this was compile with solc 0.8.15 whereas the previous was 0.8.11 I think.